[2024-01-01T16:26:55.419+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.second_task scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2024-01-01T16:26:55.426+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.second_task scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2024-01-01T16:26:55.427+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2024-01-01T16:26:55.438+0100] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2023-08-14 00:00:00+00:00
[2024-01-01T16:26:55.440+0100] {standard_task_runner.py:57} INFO - Started process 48714 to run task
[2024-01-01T16:26:55.446+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v4', 'second_task', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '12607', '--raw', '--subdir', '/Users/user/Desktop/Learning/AIRFLOW-TUTORIAL/methode-1/dags/our_first_dag.py', '--cfg-path', '/var/folders/gn/cr4yj18d1tv34pkws9v_kq1r0000gn/T/tmp6efq6p3u']
[2024-01-01T16:26:55.449+0100] {standard_task_runner.py:85} INFO - Job 12607: Subtask second_task
[2024-01-01T16:26:55.493+0100] {task_command.py:416} INFO - Running <TaskInstance: our_first_dag_v4.second_task scheduled__2023-08-14T00:00:00+00:00 [running]> on host macbook-pro-de-user.local
[2024-01-01T16:26:55.906+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aida' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-14T00:00:00+00:00'
[2024-01-01T16:26:55.907+0100] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/gn/cr4yj18d1tv34pkws9v_kq1r0000gn/T
[2024-01-01T16:26:55.908+0100] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am the second task and I will be running after the first task!']
[2024-01-01T16:26:55.916+0100] {subprocess.py:86} INFO - Output:
[2024-01-01T16:26:55.920+0100] {subprocess.py:93} INFO - hey, I am the second task and I will be running after the first task!
[2024-01-01T16:26:55.920+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-01T16:26:55.938+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=second_task, execution_date=20230814T000000, start_date=20240101T152655, end_date=20240101T152655
[2024-01-01T16:26:55.990+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-01T16:26:56.010+0100] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
